src/Magnum/Audio/Context.h

```diff
@@ -49,12 +89,19 @@ class MAGNUM_AUDIO_EXPORT Context {
         /** @brief Current context */
         static Context* current() { return _current; }
 
+        class Configuration;
+
         /**
          * @brief Constructor
          *
-         * Creates OpenAL context.
+         * Creates OpenAL context with given configuration.
          */
+        #ifdef DOXYGEN_GENERATING_OUTPUT
+        explicit Context(const Arguments& arguments, const Configuration& configuration = Configuration());
```

Comment:
This should be just `explicit Context(const Configuration& configuration = Configuration());`, I think